challenge-03/README.md

```diff
@@ -150,8 +171,28 @@ deve conter no retorno da frase acima é "metro" no lugar de "metros".
 método), que será concatenada com a frase de retorno, mostrando a resposta
 correta, de acordo com os dados inseridos no objeto.
 */
-?
+pessoa.apresentacao = function(){
+	if ( pessoa.sexo === 'masculino' ){
+		var artigo = 'o';
+	} else {
+		var artigo = 'a';
```

Comment:
-1
Não é necessário declarar as variáveis duas vezes. Você pode declarar as variáveis no início da função, e usar o `if` para alterar o valor da variável. Nesse caso, nem seria necessário o `else` ;)